tests/integration/test_data/non_self_expose_channel/recipe.yaml

```
@@ -0,0 +1,27 @@
+recipe:
+  name: expose
+  version: 1.0.0
+
+outputs:
+  # expose-non-expose the non-expose tool, while non-expose only depends on expose-non-expose
+  - package:
+      name: expose-non-expose
```

```suggestion
      name: jupyter
```

What I like about using a real name here is that it immediately motivates why we implement this functionality